### ISSUE TITLE
ONNX does not support "multi:softmax" of XgBoost

### ISSUE THREAD

**Issue by PaoloRanzi81** [onnx/onnxmltools#400]
Hi, 
first I wanted to thank you for the nice library. 

I am trying to convert XgBoost mult-nomial/multi-class logistic regression to ONNX. Checking the source code I have noticed that the file "_parse.py" does not support "multi:softmax", but only "multi:softprob". Indeed, the predictions are all logits and not the actual classes. The predictions I get with "multi:softmax" are exactly the same the ones I get by using "multi:softprob". I think the "multi:softmax" gets overridden automatically by ONNX. 

Thanks, 
Paolo

On Ubuntu 18.04, I am using: 
onnx==1.7.0
xgboost==1.1.0

**Comment by xadupre** [comment]
I almost finished. With softmax, xgboost produces label and margins, not probabilities. I'm hesitating for the second output, margins or probabilities.
